spec/02-integration/05-proxy/01-router_spec.lua

```diff
@@ -303,21 +302,23 @@ describe("Router", function()
       insert_apis {
         {
           name         = "api-strip-uri",
-          upstream_url = "http://httpbin.org",
+          upstream_url = helpers.mock_upstream_url,
           uris         = { "/x/y/z", "/z/y/x" },
           strip_uri    = true,
         },
       }
 
       helpers.run_migrations()
-      assert(helpers.start_kong())
+      assert(helpers.start_kong({
+        nginx_conf = "spec/fixtures/custom_nginx.template",
```

Comment:
I think we can include the custom Nginx configuration in all of our test instances now? That is, directly in the `kong_exec` function in `spec/helpers.lua`, since all instances started by our test suite need this custom template now, don't they?